bin/storm.cmd

```diff
@@ -90,6 +90,7 @@
   )
   set path=%PATH%;%STORM_BIN_DIR%;%STORM_SBIN_DIR%
   call start /b "%storm-command%" "%JAVA%" %JAVA_HEAP_MAX% %STORM_OPTS% %STORM_LOG_FILE% %CLASS% %storm-command-arguments%
+  exit /b %ERRORLEVEL%
   goto :eof
```

Comment:
@knusbaum Yes, I do not see any control flow issue of removing or keeping it with this change. I did not do it to keep the sub routine blocks consistent so that all end with goto :eof, plus this is an already deployed and working change in Apache Storm on Azure HDInsight for more than 6 months.